backend/typescript/services/implementations/campService.ts

```diff
@@ -68,6 +66,33 @@ class CampService implements ICampService {
         return [];
       }
 
+      if (year) {
+        // Filter out camps that have camp sessions in the desired year
+        /* eslint-disable no-param-reassign */
+        camps = camps.filter((camp) => {
+          // Go through every camp session in a camp. If it has the desired year, include it in the results.
+          /* eslint-disable no-param-reassign */
+          camp.campSessions = (camp.campSessions as CampSession[]).filter(
+            (campSession) => {
+              return campSession.dates.every((campDate) => {
+                const startCampYearTime = new Date(year, 0, 1).getTime();
```

Comment:
I printed out what date this refers to and I got `Date: "2022-01-01T05:00:00.000Z" ` do we need to set the hour to 00?

Reply:
Removed this line so shouldn't be an issue now!